cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractAssemblyOperator.java

```diff
@@ -40,22 +47,30 @@
     private static final Logger log = LogManager.getLogger(AbstractAssemblyOperator.class.getName());
 
     protected static final int LOCK_TIMEOUT = 60000;
+    protected static final int DEFAULT_CERTS_EXPIRATION_DAYS = 365;
```

Comment:
We should not define this here - we should reuse the value from `AbstractModel`, or? To have the value defined in single place only.

Reply:
I don't have any specific use case in mind but maybe could be useful to have the "general" internal CA certificates having a different expiration date from the other "cluster specific" certificates?